docs/v0.3/sdk/go/fssrz-index.md

```diff
@@ -0,0 +1,38 @@
+---
+slug: /fssrz/sdk-go
+---
+
+# Dagger SDK for Go
+
+<div class="status-badge">Technical Preview</div>
+
+Dagger is a cross-platform CI/CD engine with three defining features:
+
+1. **Portable**. Your pipelines run in containers so you get the same behavior on your local machine and in your CI environment.
+2. **Scriptable**. Develop pipelines in Go, Typescript, Python, or even a shell script. No niche language or proprietary YAML is required.
+3. **Extensible**. Each pipeline has an API. Pipelines can be shared and reused across projects, teams or the entire community.
+
+The Dagger SDK for Go enables you to communicate with the Dagger API by calling native Go functions. With this SDK, you can author your CI/CD pipeline entirely in Go, without needing to learn a new language or think about interoperability.
+
+## Usage
+
+The following sequence diagram explains how the Dagger SDK can be used in the typical CI/CD process flow:
+
+```mermaid
+graph LR;
+
+script["your Go application"] -. Dagger SDK ..-> engine["Dagger Engine"]
```

Comment:
A few important details:

1. Let's use "tool" instead of "application", will be less confusing in context (there is probably a Go application nearby, but we don't touch it)
2. Make the diagram specifically about this SDK ("your go tool", "go SDK"...)
3. In this first launch, the engine will not yet be released as a standalone component, therefore we are keeping it out of the spotlight for now. Better to have a simplified diagram that matches the simplified story, and gradually add detail as we launch more pieces.